README.md

```diff
@@ -0,0 +1,190 @@
+
+```
+Model            Controller        Cloud/Region         Version  SLA          Timestamp
+test-integrator  lxd-controller-1  localhost/localhost  2.9.34   unsupported  13:05:56Z
+
+App              Version  Status   Scale  Charm            Channel   Rev  Exposed  Message
+data-integrator           blocked      1  data-integrator             79  no       Please relate the data-integrator with the desired product
+mongodb                   active       1  mongodb          dpe/edge   99  no       
+
+Unit                 Workload  Agent  Machine  Public address  Ports      Message
+data-integrator/79*  blocked   idle   99       10.91.92.137               Please relate the data-integrator with the desired product
+mongodb/3*           active    idle   100      10.91.92.227    27017/tcp  
+
+Machine  State    Address       Inst id          Series  AZ  Message
+99       started  10.91.92.137  juju-554175-99   jammy       Running
+100      started  10.91.92.227  juju-554175-100  focal       Running
+
+Relation provider                      Requirer                               Interface              Type  Message
+data-integrator:data-integrator-peers  data-integrator:data-integrator-peers  data-integrator-peers  peer
+mongodb:database-peers                 mongodb:database-peers                 mongodb-peers          peer  
+
+```
+
+
+Relate the two applications with:
+```shell
+juju relate data-integrator mongodb
+```
+Wait for `watch -c juju status --relations --color` to show:
+```
+Model            Controller        Cloud/Region         Version  SLA          Timestamp
+test-integrator  lxd-controller-1  localhost/localhost  2.9.34   unsupported  13:06:58Z
+
+App              Version  Status  Scale  Charm            Channel   Rev  Exposed  Message
+data-integrator           active      1  data-integrator             79  no 
+mongodb                   active      1  mongodb          dpe/edge   99  no 
+
+Unit                 Workload  Agent  Machine  Public address  Ports      Message
+data-integrator/79*  active    idle   99       10.91.92.137                   
+mongodb/3*           active    idle   100      10.91.92.227    27017/tcp  
+
+Machine  State    Address       Inst id          Series  AZ  Message
+99       started  10.91.92.137  juju-554175-99   jammy       Running
+100      started  10.91.92.227  juju-554175-100  focal       Running
+
+Relation provider                      Requirer                               Interface              Type     Message
+data-integrator:data-integrator-peers  data-integrator:data-integrator-peers  data-integrator-peers  peer
+mongodb:database                       data-integrator:mongodb                mongodb_client         regular
+mongodb:database-peers                 mongodb:database-peers                 mongodb-peers          peer
+```
+To retrieve information such as the username, password, and database. Enter:
+```shell
+juju run-action  data-integrator/leader get-credentials --wait
```

Comment:
**todo:** There is a double-space between `run-action` and `data-integrator`. I am not certain how the Juju CLI handles this, but just in case someone copies and pastes this, I'd remove the double-space.

Reply:
The extra space is handled by the Juju CLI but I will remove it